Source/WebKit/UIProcess/WebProcessPool.cpp

```diff
@@ -1121,10 +1120,19 @@ Ref<WebPageProxy> WebProcessPool::createWebPage(PageClient& pageClient, Ref<API:
         pageConfiguration->setWebsiteDataStore(WebKit::WebsiteDataStore::defaultDataStore().ptr());
     }
 
+    auto haveSameGPUProcessPreferences = [pageConfiguration](const WebPageProxy& relatedPage) {
+#if ENABLE(GPU_PROCESS)
+        return relatedPage.preferencesForGPUProcess() == pageConfiguration->preferencesForGPUProcess();
```

Comment:
Can this use `hasSameGPUProcessPreferencesAs()`?

In general, there are lot of places in this PR where you compare `preferencesForGPUProcess()`; it would be nice if they all went through some bottleneck function so they are easy to find.